sbpy/bib/tests/test_bib.py

```diff
@@ -144,9 +150,50 @@ def force(mass, accelleration):
     reset()
     track()
     force(1, 2)
-    assert (set(['sbpy.bib.tests.test_bib.force:', 'method:',
-                 '1687pnpm.book.....N'])
-            == set(show().split()))
+    assert (set(
+        ['sbpy.bib.tests.test_bib.test_cite_function.<locals>.force:',
+         'method:', '1687pnpm.book.....N'])
+        == set(show().split()))
+    stop()
+    reset()
+
+
+def test_cite_class_method():
+    reset()
+
+    class Physics:
+        @cite({'method': '1687pnpm.book.....N'})
+        def force(self, mass, accelleration):
```

Comment:
`force` could be a static method but not important because it is only used for testing.

Reply:
Got it.